### DISCUSSION TITLE
[Feature]: Group support warp like Photoshop

### DISCUSSION THREAD

**Discussion by zzq0324** [fabricjs/fabric.js#10055] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

I try to apply warp effect from photoshop to canvas.

I have 16 mesh points from reading psd file, but i dnt know how to apply warp effect to image from these points. It's a smart object layer, i wll convert this layer to fabric.Group. Anyone help me? Thank you!

<img width="490" alt="image" src="https://github.com/fabricjs/fabric.js/assets/24889976/dc72ec79-7daf-46b7-aeef-7adbd5f9917c">

### Current State

none

**Comment by asturur** [comment]
I m pretty much sure that starting from this codepen here
https://codepen.io/jorgalga/pen/jYJGRy
something can be done.

**Reply by zzq0324** [reply]
> I m pretty much sure that starting from this codepen here https://codepen.io/jorgalga/pen/jYJGRy something can be done.

like this is very cool, it use pixi.js  [https://codepen.io/osublake/pen/gLGYxx](https://codepen.io/osublake/pen/gLGYxx)
